crates/bevy_text/src/text.rs

```diff
@@ -1,15 +1,35 @@
 use bevy_asset::Handle;
 use bevy_color::Color;
+use bevy_derive::{Deref, DerefMut};
 use bevy_ecs::{prelude::Component, reflect::ReflectComponent};
 use bevy_reflect::prelude::*;
 use bevy_utils::default;
+use cosmic_text::{Buffer, Metrics};
 use serde::{Deserialize, Serialize};
 
 use crate::Font;
+pub use cosmic_text::{
+    self, FamilyOwned as FontFamily, Stretch as FontStretch, Style as FontStyle,
+    Weight as FontWeight,
+};
 
+/// Wrapper for [`cosmic_text::Buffer`]
+#[derive(Component, Deref, DerefMut, Debug, Clone)]
+pub struct CosmicBuffer(pub Buffer);
+
+impl Default for CosmicBuffer {
+    fn default() -> Self {
+        Self(Buffer::new_empty(Metrics::new(0.0, 0.000001)))
```

Comment:
If this is still necessary (no idea) it could use a comment.

Reply:
I presume these metrics are always overwritten. But even so, I wonder if we'd be better off with a sensible default font size rather than these zero sizes.

Reply:
Seems like `Metrics::default()` would be alright. Agree that they will always be overwritten by metrics in spans.